src/tink/anon/Macro.hx

```diff
@@ -45,22 +45,28 @@ class Macro {
     });
   }
   
-  static public function mergeExpressions(exprs:Array<Expr>, ?requiredType, ?pos, ?as) {
+  static public function mergeExpressions(exprs:Array<Expr>, ?requiredType, ?pos, ?as, accept_first=false) {
```

Comment:
here's where mergeExpressions adds the additional parameter.  We're telling to always accept the first occurrence of a field as the override.  Subsequent field names in another anonymous types are ignored.